recoco/apps/survey/migrations/0034_question_slug.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 4.2.13 on 2024-06-17 10:12
+
+import autoslug.fields
+from django.db import migrations
+import recoco.apps.survey.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("survey", "0033_alter_session_project"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="question",
+            name="slug",
+            field=autoslug.fields.AutoSlugField(
+                always_update=True,
+                blank=True,
+                editable=False,
+                null=True,
+                populate_from=recoco.apps.survey.models.Question._populate_slug,
+                unique=True,
```

Comment:
`always_update`, `null` and `blank` are set to True only for migration purpose.
1. create the slug field
2. fill the values (`always_update` is helping here)
3. set the definitive field params